mobile/src/main/java/org/openhab/habdroid/ui/ViewExtensions.kt

```diff
@@ -120,14 +121,19 @@ fun RemoteViews.duplicate(): RemoteViews {
 }
 
 fun MaterialButton.setTextAndIcon(connection: Connection, mapping: LabeledValue) {
-    contentDescription = mapping.label
-    val iconUrl = mapping.icon?.toUrl(context, true)
+    setTextAndIcon(connection, mapping.label, mapping.icon)
```

Comment:
No need for this method overload, just give `icon` a default value below.

Reply:
I'm not sure I understand what you meant. Would you mind elaborating on this please?

Reply:
You currently have two `setTextAndIcon` methods with different parameters, where one redirects to the other. You don't need the 2 parameter version, as you can implement the 3 parameter version with a default parameter for `icon`:
`fun MaterialButton.setTextAndIcon(connection: Connection, label: String, icon: IconResource? = null)`

Reply:
The 2 parameter version takes a `LabeledValue`, not a String. I can't see a clean way to combine them into one.

Reply:
I can go and change all the calls to setTextAndIcon to use the 3 parameter version. Perhaps that's better anyway.

Reply:
> The 2 parameter version takes a LabeledValue, not a String

Oh, I missed that during review. Adjusting the calls is the right solution :+1: